### ISSUE TITLE
Help text for islandora_datastream_crud_generate_derivatives describes deleting datastreams

### ISSUE THREAD

**Issue by kayakr** [SFULibrary/islandora_datastream_crud#60]
`islandora_datastream_crud_generate_derivatives` has example
` 'drush islandora_datastream_crud_delete_datastreams --user=admin --dsid=FOO --pid_file=/tmp/delete_foo_from_these_objects.txt',`
and help text like 
`'pid_file' => array(
        'description' => 'Absolute path to the file that lists PIDs for objects you want to delete datastreams from.',`

**Comment by mjordan** [comment]
Thanks, I'll fix.

**Comment by mjordan** [comment]
Running `drush islandora_datastream_crud_generate_derivatives --help` now produces:

```
Generate or regenerate derivative datastreams from a source DSID.

Examples:
 0                                         drush islandora_datastream_crud_generate_derivatives --user=admin              
                                           --source_dsid=OBJ --pid_file=/tmp/regenerate_derivatives_for_these_objects.txt

Options:
 --datastreams_crud_log                    Optional. Absolute path to the CRUD log. If present, the PID, DSID, and a      
                                           message will be written to the specified file.                                 
 --dest_dsids                              Optional. A comma-separated list of datastream IDs to generate. If absent, all 
                                           derivatives will be generated.                                                 
 --pause                                   Optional. Number of seconds to pause between objects.                          
 --pid_file                                Absolute path to the file that lists PIDs for objects you want to generate     
                                           derivatives for. Required.                                                     
 --skip_dsids                              Optional. A comma-separated list of datastream IDs to skip. If absent, all     
                                           derivatives will be generated.                                                 
 --source_dsid                             The source datastream ID. Required.

Aliases: idcrudgd
```

**Comment by mjordan** [comment]
Addressed with 40faa8348840f991d6207d115ebff1e0d1d33bc6.
